### DISCUSSION TITLE
【微信图片版权】”此图片来自微信公众平台，未经允许不可引用“

### DISCUSSION THREAD

**Discussion by dadaoerv** [tangly1024/NotionNext#2063] [General]
**为什么提出这个新的特性改动**
剪藏的笔记，在notion里显示是正常的，但是作为博客发布就不行了。

**描述一下你推荐的解决方案**
https://developers.weixin.qq.com/community/minihome/doc/000e625b3acac8c85d0ded78e5bc00
解决方法：在<head>中加入这一句meta name="referrer" content="never"

**描述一下你考虑过的其它替代解决方案**
那么问题是怎么在notionnext中解决

**补充说明**
补充与此特性相关的内容

**Comment by tangly1024** [comment]
据我了解，微信的图片只能在腾讯系的域名下使用。

## notion和网站区别：
1. notion中： 微信图片，仅在你自己的设备查看，这是允许的。
2. 在网站中：你盗用了微信的图片链接、在自己的域名进行二次发布，构成侵权。

## 一个小实验：
>  剪藏的笔记，在notion里显示是正常的，但是作为博客发布就不行了。

将这个notion页面网址直接共享给别人查看，看是否能够正常显示。

## 解决方案：
1. 将微信公众号内的图片下载，在notion中上传，从而保证图片地址非微信域名。
2. 使用一键剪藏的工具，可以开启图床功能，将自动下载微信图片，并上传到自定义图床。
